src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -238,7 +244,8 @@ TER PathCursor::deliverNodeReverse (
             resultCode = increment(-1).deliverNodeReverse(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                /*restart dir*/false);
```

Comment:
OK - it didn't work out as neatly as I'd have liked.  My intention when I worked on this code was that these methods would become free functions on a struct (c.f. `RPC::Context`) but they seem to have become more method-y, not less.

https://github.com/rec/rippled/commit/f6591fde7047f73b9652c2477acb8f34d0f75b42

Doesn't work either on npm tests, I did something wrong.  Not really worth doing I think.  If we could hide the existence of the impl in the .cpp it would.

Worth a try!  :-)